_data/tools.yml

```diff
@@ -981,6 +981,27 @@
       url: https://www.openapis.org/
     - urltext: Trino OpenAPI connector
       url: https://github.com/nineinchnick/trino-openapi
+- name: Open Policy Agent
+  anchor: opa
+  category: add-on
+  logo: /assets/images/logos/opa.png
+  logosmall: /assets/images/logos/opa-small.png
+  description: |
+    Open Policy Agent (OPA) is a system for policy-based control for cloud
+    native environments. It enables flexible, fine-grained control for
+    administrators across many systems and applications.
+
+    The Trino plugin enables the use of Open Policy Agent (OPA) as authorization
+    engine for access control to catalogs, schemas, tables, and more in Trino.
```

Comment:
Not a huge fan of "and more" language in most contexts

Reply:
Updated